dot/config_test.go

```diff
@@ -15,30 +15,3 @@
 // along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
 
 package dot
```

Comment:
should we remove this file?

Reply:
I kept this file to add unit tests to in another PR.